lib/OpenQA/Schema/Result/Assets.pm

```diff
@@ -215,6 +219,8 @@ sub limit_assets {
     if (opendir($dh, $OpenQA::Utils::assetdir . "/repo")) {
         my %repos;
         while (readdir($dh)) {
+            next if $_ eq '.' || $_ eq '..';
```

Comment:
Not sure what you smoked that night, but code length is not a goal I optimize for